pkg/asset/manifests/network.go

```diff
@@ -126,10 +95,15 @@ func (no *Networking) Generate(dependencies asset.Parents) error {
 		return errors.Wrapf(err, "failed to create %s manifests from InstallConfig", no.Name())
 	}
 
+	crdContents := ""
+	for _, crdFile := range crds.Files() {
+		crdContents = fmt.Sprintf("%s\n---\n%s", crdContents, crdFile.Data)
```

Comment:
Why do we want to put these two into the same file rather than keeping them as two separate manifests? If there is a compelling reason to combine them into the same manifest, then why are they separated in the manifest template?

Reply:
Because our `yamllint` configuration doesn't allow for multiple documents in the same template file. There's zero reason for it to be separate files otherwise. It is the two CRDs necessary for creating network configuration.